app/models/mongoidable/policy_relation.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Mongoidable
```

Comment:
An embedded model that works as a has policy through type relation. It contains the required attributes in order to apply the policy to the given model.